machineSetup/setupPerfMachine.ps1

```diff
@@ -0,0 +1,144 @@
+﻿#Set power scheme to high performance
+powercfg /S 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c
```

Comment:
These machines should never be used for anything other than performance runs so I am not too worried about over-writing the default `SCHEME_MIN`